index.js

```diff
@@ -99,6 +111,9 @@ class KinesisStreamReader extends Readable {
         if (this.options.filter.test(record)) {
           this.push(record)
         }
+        if (endTimestamp && JSON.parse(record).metadata.timestamp > endTimestamp) {
```

Comment:
lets aim to publish this repo. instead of the hardcoding the timestamp field, can we get either from the kinesis metadata or through user specified option of specifying the field name

Reply:
this 'endTimestamp' is user specified option